backend/apps/evaluation/templates/evaluation/studentevaluation_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load i18n %}
+
+{% block title %}
+Evaluation Studis
```

Comment:
Since you said you were new to django I am going to add that all string literals have to be translated and thus need to be surrounded with either `{% blocktrans %}string literal here{% endblocktrans %}` or `_("string literal here")`. Also refer to the django i18n guide: https://docs.djangoproject.com/en/3.0/topics/i18n/translation/
The correct way here would be (you can directly commit this here):
```suggestion
{% blocktrans %}Evaluation Studis{% endblocktrans %}
```

Reply:
and then add the translation to `backend/locale/en/LC_MESSAGES/django.po` after running the `makemessage` command